### ISSUE TITLE
[bug] kserving-component still uses minio however boto3 is officially added in kf-serving to use IRSA

### ISSUE THREAD

**Issue by ahsan-n** [kubeflow/pipelines#5700]
### What steps did you take

Hi, so i am creating pipeline to host / serve model that resides in s3. For that I used [example for sample pipeline](https://github.com/kubeflow/kfserving/blob/master/docs/samples/pipelines/sample-tf-pipeline.py) which has [kfserving-component ](https://raw.githubusercontent.com/kubeflow/pipelines/master/components/kubeflow/kfserving/component.yaml). I replaced gc path with my s3 bucket.

### What happened:
I get the following error in storage

```
[I 210519 20:13:05 initializer-entrypoint:13] Initializing, args: src_uri [s3://my-bucket/saved_model] dest_path[ [/mnt/models]
[I 210519 20:13:05 storage:45] Copying contents of s3://my-bucket/saved_model to local
Traceback (most recent call last):
  File "/storage-initializer/scripts/initializer-entrypoint", line 14, in <module>
    kfserving.Storage.download(src_uri, dest_path)
  File "/usr/local/lib/python3.7/site-packages/kfserving/storage.py", line 62, in download
    Storage._download_s3(uri, out_dir)
  File "/usr/local/lib/python3.7/site-packages/kfserving/storage.py", line 89, in _download_s3
    for obj in objects:
  File "/usr/local/lib/python3.7/site-packages/minio/api.py", line 2297, in _list_objects
    query=query,
  File "/usr/local/lib/python3.7/site-packages/minio/api.py", line 2229, in _url_open
    object_name).get_exception()
minio.error.AccessDenied: AccessDenied: message: Access Denied

```

### What did you expect to happen:
When looking over github i found out that because minio does not support iam role for service accounts, [boto3 has replaced minio](https://github.com/kubeflow/kfserving/pull/1472). Now I am wondering how can i use that kfserving-component but with boto3 :) 
### Environment:

kubeflow v1.3

*  How do you deploy Kubeflow Pipelines (KFP)? 
use kfp.Compile to extract out yaml and uploaded in pipelines UI

*  KFP SDK version: 
v1.6.2

### Labels

 /area components 

---

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

**Comment by rawc0der** [comment]
@ahsan-n, I think the problem is solved in next release `0.6` of kfserving.

But the docker image `quay.io/aipipeline/kfserving-component:v0.6.0` is not yet available. Think this still needs some attention to confirm it's been resolved.

**Comment by ahsan-n** [comment]
Hi @rawc0der  just wondering what is the release timeline for `quay.io/aipipeline/kfserving-component:v0.6.0` ?
